kolibri/utils/options.py

```diff
@@ -352,3 +352,22 @@ def update_options_file(section, key, value, KOLIBRI_HOME, ini_filename="options
             file=conf.filename
         )
     )
+
+
+def generate_empy_options_file(options_path, options_data):
+
+    exclude_key = ["Python"]
+
+    # Generate an options.ini file inside the KOLIBRI_HOME as default placeholder config
+    with open(options_path, "w") as file:
+        for k, v in options_data.items():
+            if k in exclude_key:
+                return
```

Comment:
I think `continue` would be better here since `options_data` is a dictionary, which means `Python` key may not always come at the end. 
When I tested locally, the program read `Python` as the third key, and exited after that.

Reply:
it's often more pythonic to work with dictionaries like this:

```python
    keys = [k for k in options_data if k not in excluded_keys]
    for key in keys:
            file.write("# [{}] \n".format(key))
            child_keys = [k for k in options_data[key] if k not in excluded_keys]
            for child_key in child_keys:
                file.write("# {} = {} \n".format(child_key, options_data[key][child_key]))
            file.write("\n")    
```